include/constants/flags.h

```diff
@@ -1350,7 +1350,7 @@
 #define FLAG_SYS_POKEMON_GET                         (SYSTEM_FLAGS + 0x0) // FLAG_0x860
 #define FLAG_SYS_POKEDEX_GET                         (SYSTEM_FLAGS + 0x1)
 #define FLAG_SYS_POKENAV_GET                         (SYSTEM_FLAGS + 0x2)
-#define FLAG_UNUSED_0x863                            (SYSTEM_FLAGS + 0x3) // Unused Flag
+#define FLAG_SYS_POKENAV_FLY                         (SYSTEM_FLAGS + 0x3) // Used to tell the game when the user is flying from their Pokenav, so the "Show Mon" animation doesn't play. Should be cleared by the time the player lands.
```

Comment:
We don't touch regular flags and handle everything through define to which a user ca assign their own flags.

Reply:
no, you shouldn't touch flags at all. It should be attached to a define.

Reply:
fixed and working correctly!